KalturaNetKit.podspec

```diff
@@ -1,6 +1,6 @@
 
-suffix = '-dev'   # Dev mode
-# suffix = ''       # Release
+# suffix = '-dev'   # Dev mode
+suffix = ''       # Release
```

Comment:
is this intentional?

Reply:
Yes, see my comment in the beginning.

_"Set to release mode, otherwise QA won't be able to check, need to fix PlayKit dependency to it."_

Reply:
Missed it, thanks.